src/templates/content.html

```diff
@@ -40,6 +40,9 @@ <h1>Video Index</h1>
         <p>Video ID: {{ video.video_id }}</p>
         <p>Video Extension: {{ video.extension }}</p>
         <p>User ID: {{ video.user_id }}</p>
+        <a href="/delete/{{ video.video_id }}.{{ video.extension }}">
```

Comment:
In this page, you can also access the `video.user_id` property. In the `app.py` route, we should also pass `current_user.user_id` as a variable to this page as `current_user_id` or similar. Then put an `if` statement around this button to only display it if `video.user_id` matches `current_user_id` or whatever you name the variable.

Reply:
Thinking about this again, it might be the case that `current_user.user_id` is actually the username and not the user's UUID. It might be necessary to get the UUID from that username and pass that to the page as `current_user_id` instead.

Reply:
yup. I agree

Reply:
It worked! I added to the / route a variable "signed_in" for the user currently signed in. It will compare that to user_id associated with a video. Only those that match will get the button.